문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+    let answer = '';
+    return s.split('').sort().reverse().join('');
```

Comment:
잘 푸셨는데 answer를 선언한 이유는 뭔가요?
answer 변수를 어느 곳에도 사용하지 않고 있어요..
사용하지 않는 변수는 지우는게 좋겠죠?

```suggestion
    return s.split('').sort().reverse().join('');
```